### PR TITLE
Bunch of updates

### LINKED ISSUE #110: Add other process runners to install guide
Now supports overmind, forego, heroku

### PR DESCRIPTION
**Pull request by paulcsmith**
Closes #110

### CODE REVIEW

File: source/guides/actions-and-routing.html.md.erb [hunk 1]
```diff
@@ -368,3 +368,60 @@ class FooAction < BrowserAction
...
+```
+# Typically used when calling `render` instead of `redirect`
+# That way the message shows for the current request only
+flash.now.success = "Woo-hoo!"
+```
+
+Or sometimes you want to keep all messages for the next request:
+
+```
+# Useful when you want to redirec to another page while keeping the flash messages
```
Comment (andyw8) [inline source/guides/actions-and-routing.html.md.erb]: redirect

File: source/guides/actions-and-routing.html.md.erb [hunk 2]
```diff
@@ -368,3 +368,60 @@ class FooAction < BrowserAction
...
 ```
+
+## Flash messages
+
+You can show messages using `flash`. This works similarly to Rails, Phoenix,
+and other frameworks.
+
+### Type safe flash messages
+
+The built in messages are `success`, `danger` and `info`. Using these will
```
Comment (andyw8) [inline source/guides/actions-and-routing.html.md.erb]: Perhaps "message variants" or "message types", so as to distinguish from the message string itself?

File: source/guides/actions-and-routing.html.md.erb (hunk 1 above)
Comment (paulcsmith) [reply source/guides/actions-and-routing.html.md.erb]: Whoops :)

File: source/guides/actions-and-routing.html.md.erb (hunk 2 above)
Comment (paulcsmith) [reply source/guides/actions-and-routing.html.md.erb]: I like it!

**Pull request merged**